java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -3056,9 +3057,9 @@ void testConcatListsStrings() {
          ColumnVector expected = ColumnVector.fromLists(new HostColumnVector.ListType(true,
              new HostColumnVector.BasicType(true, DType.STRING)) , list, list3, list2)) {
       assert res1.getNullCount() == 1: "Null count is incorrect on input column";
-      assert res1.getChildColumnViewAccess(0).getNullCount() == 0 : "Null count is incorrect on input column";
+      assert res1.getChildColumnView(0).getNullCount() == 0 : "Null count is incorrect on input column";
```

Comment:
These need to be closed too or we are leaking.  It was a bug before, but I think we should fix it.